features/raadpleeg-met-reisdocumentnummer/gba/autorisatie-gba.feature

```diff
@@ -0,0 +1,132 @@
+# language: nl
+
+@gba @autorisatie
+Functionaliteit: autorisatie voor het gebruik van de API RaadpleegMetReisdocumentnummer
+  Autorisatie voor het gebruik van de API gebeurt op twee niveau's:
+  1. autorisatie van de gebruiker door de afnemende organisatie
+  2. autorisatie van de afnemer (organisatie) door RvIG
+
+  Deze feature beschrijft alleen de autorisatie van de afnemer door RvIG.
+
+  Voorlopig wordt de reisdocumenten bevragen API alleen aangeboden aan gemeenten voor het raadplegen van reisdocumenten van eigen inwoners.
+  Gebruikers zijn daarom verplicht om de parameter gemeenteVanInschrijving te gebruiken en mogen dan alleen de gemeentecode van de eigen gemeente gebruiken.
+
+  Autorisatie wordt verkregen met behulp van een OAuth 2 token. 
+  In het verkregen token is de afnemerindicatie opgenomen die de afnemer uniek identificeert. Op basis van de afnemerindicatie kan de autorisatie worden opgezocht.
+  Wanneer de afnemer een gemeente is, is er ook een gemeentecode opgenomen in de OAuth token.
+
+  Voor één afnemer kunnen er meerdere rijen zijn in de autorisatietabel, maar daarvan kan er maar één actueel zijn. Alleen de actuele mag worden gebruikt.
+  Een autorisatie is actueel wanneer de Datum ingang (35.99.98) in het verleden ligt en Datum beëindiging tabelregel (35.99.99) leeg is of in de toekomst ligt.
+
+  # To Do (t.z.t.): regels voor autorisatie op basis van een autorisatiebesluit
+  # To Do (t.z.t.): regels voor leveren persoonsgegevens met geheimhouding aan derde (35.95.12)
+  # To Do (t.z.t.): regels voor voorwaarderegel ad hoc (35.95.61)
+
+
+
+    Achtergrond:
+      Gegeven de persoon met burgerservicenummer '000000152' heeft een 'reisdocument' met de volgende gegevens
+      | naam                                                 | waarde    |
+      | soort reisdocument (35.10)                           | PN        |
+      | nummer reisdocument (35.20)                          | NE3663258 |
+      | datum einde geldigheid reisdocument (35.50)          | 20240506  |
+      | datum inhouding/vermissing reisdocument (35.60)      | 20230405  |
+      | aanduiding inhouding/vermissing reisdocument (35.70) | I         |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | gemeente van inschrijving (09.10) |
+      | 0800                              |
+
+
+  Rule: Een gemeente als afnemer is geautoriseerd voor alle zoekvragen en alle gegevens voor haar eigen inwoners
```

Comment:
Zullen we niet gewoon benoemen waarvoor de afnemer is geautoriseerd? Raadplegen op rdsnr en zoeken op BSN, en de reisdocgegevens benoemen, zodat als er iets bijkomt we expliciet moeten benoemen wat daarvoor geldt?

Reply:
Zoals vanmorgen uitgelegd: dit blijft geen onderdeel van de BRP. Vandaar de overweging. Een aanpassing is niet nodig.